Library/SharedFunctions.swift

```diff
@@ -137,13 +150,40 @@ public func countdownProducer(to date: Date)
       .map(formattedComponents(dateComponents:))
 }
 
-private func formattedComponents(dateComponents: DateComponents)
-  -> (day: String, hour: String, minute: String, second: String) {
+extension LiveStreamEvent {
```

Comment:
did you consider putting this in `LiveStreamEvent`? This uncategorized `SharedFunctions` file feels a bit out of place with our tight organization, ya know?